src/Request/ShowCard.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Request;
+
+class ShowCard
```

Comment:
After your changes, we could get rid of all those fields and leave only `$carId`